server-function/shift_deploy_binaris.sh

```diff
@@ -1,23 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
(Just a question, I don't understand)

Why did we need this field, but not any more?

Reply:
It was re-implemented in https://github.com/binaris/shiftjs/pull/138, so this is getting rid of the old implementation to avoid confusion.